doc/developer/debugging.rst

```diff
@@ -164,11 +164,24 @@ Just define an environment variable in the build:
 Developing in Python
 --------------------
 
-Add the following environment variable to the geoportal container:
+Create a development docker-compose.overide.yml
+...............................................
 
-``GUNICORN_PARAMS="-b :80 --worker-class gthread --threads 1 --workers 1 --reload"``
+Add a ``docker-compose.override.yml`` file with a ``geoportal`` service containing the following lines:
+
+.. code:: yaml
+
+   ---
+
+   version: '2'
 
-TODO
+   services:
+     geoportal:
+       volumes:
+         - ${PWD}/geoportal/<project>_geoportal:/app/aprona_geoportal
+       environment:
+         - GUNICORN_CMD_ARGS=--reload
```

Comment:
Shouldn't we add `-b :8080` to be on the right port?

Reply:
No, as I do not overwrite the c2cwsgiutils GUNICORN_PARAMS that is applied on the command line when running gunicorn.
This is the native gunicorn environment variable that allow to add parameters using environment variables, I found this method less intrusive with the base parameters given for production in GUNICORN_PARAMS.

Reply:
+1, I miss that you use a different environment variable :-)

Reply:
Note that I do not apply "--threads 1 --workers 1" anymore, but this seems optional for me.

Reply:
I get some issue with that, then it's good to remove that :-)